src/docs/truffle/reference/truffle-commands.md

```diff
@@ -213,12 +199,11 @@ truffle install <package_name>[@<version>]
 
 Options:
 
-* `<package_name>`: Name of the package as listed in the Ethereum Package Registry. (required)
-* `@<version>`: When specified, will install a specific version of the package, otherwise will install the latest version.
+- `<package_name>`: Name of the package as listed in the Ethereum Package Registry. (required)
+- `@<version>`: When specified, will install a specific version of the package, otherwise will install the latest version.
 
 See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
```

Comment:
Same as above.

Reply:
Thanks! Going to replace with: https://trufflesuite.com/docs/truffle/getting-started/package-management-via-ethpm.html